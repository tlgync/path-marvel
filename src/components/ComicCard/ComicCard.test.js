import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { ComicCard } from '.';

test('ComicCard Render SnapShot', () => {
  const title = 'Avengers: The Initiative (2007) #19';
  const path = 'http://i.annihil.us/u/prod/marvel/i/mg/c/e0/535fecbbb9784';
  const pageCount = '32';
  const date = new Date('2008-12-17T00:00:00-0500').toLocaleDateString();
  const price = '2.99';
  const tree = renderer.create((
    <Router>
      <ComicCard title={title} path={path} pageCount={pageCount} date={date} price={price} />
    </Router>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
