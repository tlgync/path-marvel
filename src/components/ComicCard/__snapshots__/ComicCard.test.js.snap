// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComicCard Render SnapShot 1`] = `
Array [
  <span
    className=" lazy-load-image-background blur"
    style={
      Object {
        "backgroundImage": "",
        "backgroundSize": "",
        "display": "inline-block",
        "height": 400,
        "width": 300,
      }
    }
  >
    <span
      className=""
      style={
        Object {
          "display": "inline-block",
          "height": 400,
          "width": 300,
        }
      }
    />
  </span>,
  <div
    className="comic-content"
  >
    <div
      className="comic-title"
    >
      <span>
        Avengers: The Initiative (2007) #19
      </span>
    </div>
    <div
      className="comic-title-row"
    >
      <span>
        Page Count
        : 
      </span>
      <span>
        32
      </span>
    </div>
    <div
      className="comic-title-row"
    >
      <span>
        Publish Date
        : 
      </span>
      <span>
        12/17/2008
      </span>
    </div>
    <div
      className="comic-title-row"
    >
      <span>
        Price
        : 
      </span>
      <span>
        2.99
      </span>
    </div>
  </div>,
]
`;
